api/src/routes/init.ts

```diff
@@ -26,7 +26,7 @@ export async function init(config: IConfiguration): Promise<string[]> {
 
         const currencyService = new CurrencyService(config);
         if (currencyService) {
-            log += await currencyService.update(true);
```

Comment:
Do we need to change the remaining `log+=` in this file and also in `dynamoDbService.ts`?